packages/react-select/src/components/Placeholder.tsx

```diff
@@ -29,11 +29,9 @@ export const placeholderCSS = <
 }: PlaceholderProps<Option, IsMulti, Group>): CSSObjectWithLabel => ({
   label: 'placeholder',
   color: colors.neutral50,
+  gridArea: '1 / 2',
```

Comment:
![image](https://user-images.githubusercontent.com/15950630/131299022-3d7e066a-8950-4b4b-a44e-e30a1e694fa2.png)

Forgot to provide the `Placehholder` component with the latest fix :)